usr/src/cmd/zfs/zfs_main.c

```diff
@@ -5596,17 +5596,23 @@ print_holds(boolean_t scripted, size_t nwidth, size_t tagwidth, nvlist_t *nvl)
 			uint64_t val = 0;
 			time_t time;
 			struct tm t;
-			char sep = scripted ? '\t' : ' ';
-			size_t sepnum = scripted ? 1 : 2;
+			char sep = ' ';
+			size_t sepnum = 2;
```

Comment:
It doesn't look like these are needed; you can instead use the simpler printf format of `"%-*s  %-*s  %s"`